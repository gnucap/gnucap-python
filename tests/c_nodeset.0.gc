* Felix Salfelder 2019
* GPLv3+

attach public c_python.so
loadpy c_nodeset.py

options numdgt=10

verilog

resistor #(.r(1)) r1(0,1);
capacitor #(.c(1)) c2(0,1);
nodesetter #(.extra(19), .v_1(3.3), .v_2(5), .v_a(3)) nodesetter(1, 2, a);

print dc v(nodes) iter(0)
print tran v(nodes) iter(0)

* does not work, see s_tr_swp.cc+83 ?
* transient 1 uic

* but this does
dc
transient 1

list
nodeset
nodeset v_x=5 v_y=6
nodeset

end
