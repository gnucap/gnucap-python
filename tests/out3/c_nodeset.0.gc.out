Gnucap : The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2013, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
main version
core-lib version
default plugins
generator: already installed, replacing
stashing as generator:0
transient: already installed, replacing
stashing as transient:0
transient: already installed, replacing
stashing as transient:0
#                v(1)            v(2)            v(a)            iter(0)        
 0.              3.299967        5.              3.              3.             
#Time            v(1)            v(2)            v(a)            iter(0)        
 0.              3.299967        5.              3.              2.             
 0.01            3.26729406      0.              0.              3.             
 0.0925          3.008420821     0.              0.              3.             
 0.395           2.217934568     0.              0.              3.             
 1.              1.187723118     0.              0.              3.             
resistor #(.r(1)) r1 (.p(0),.n(1));
capacitor #(.c(1)) c2 (.p(0),.n(1));
nodesetter #(.extra(19),.v_1(3.3),.v_2(5),.v_a(3)) nodesetter (.p0(1),.p1(2),.p2(a));
//* does not work, see s_tr_swp.cc+83 ? 
//* transient 1 uic 
//* but this does 
nodeset v_1= 3.3 v_2= 5. v_a= 3.
nodeset v_1= 3.3 v_2= 5. v_1= 3.3 v_2= 5. v_a= 3. v_x=NA( NA) v_y=NA( NA)
